sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -48,11 +49,10 @@ matrix([[true, false]] as const, async (useAad) => {
     beforeEach(async function (this: Context) {
       recorder = await createRecorder(this.currentTest);
       await recorder.setMatcher("BodilessMatcher");
-      client = new DocumentAnalysisClient(
-        endpoint(),
-        makeCredential(useAad),
-        recorder.configureClientOptions({})
-      );
+      client = new DocumentAnalysisClient(endpoint(), makeCredential(useAad), {
+        ...recorder.configureClientOptions({}),
+        audience: getAudience(),
```

Comment:
There are several more instances of this pattern that can be replaced with just `recorder.configureClientOptions({ audience: getAudience() }};`

Reply:
yup, thanks.